controllers/podchaos/podfailure/types.go

```diff
@@ -18,6 +18,8 @@ import (
 	"errors"
 	"fmt"
 
+	"k8s.io/client-go/util/retry"
```

Comment:
Because there is a "k8s.io/client-go/tools/cache" below, we can sort out all the "imports" here.

Reply:
done